recipes/mne-gui-addons/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "mne-gui-addons" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mne-gui-addons-{{ version }}.tar.gz
+  sha256: 0f5231065e823f051d9df09077b866cf27bda91a4582b280dfeaa725550c4ab7
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - setuptools >=45
+    - setuptools-scm >=6.2
+    - wheel
+    - pip
+  run:
+    - python >=3.8
+    - packaging
+    - qtpy
+    - pyqt
+    - pyvista
+    - pyvistaqt
+    - matplotlib-base
+    - mne-base
+    - nibabel
+    - dipy >=1.4
+    - traitlets
+    - setuptools >=65
+
+test:
+  imports:
+    - mne_gui_addons
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/mne-gui-addons/
+  summary: MNE-Python GUI addons.
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - larsoner
+    - alexrockhill
```

Comment:
@alexrockhill can you confirm that you're willing to co-maintain this recipe?

Reply:
Yes